modules/codefresh/Makefile.notification

```diff
@@ -0,0 +1,44 @@
+codefresh/notify/slack/build:
+	$(call assert-set,PIPELINE_ACTION)
+	$(call assert-set,CHART_NAME)
+	$(call assert-set,CF_BUILD_URL)
+	$(call assert-set,CF_REPO_OWNER)
+	$(call assert-set,CF_REPO_NAME)
+	$(call assert-set,CF_BUILD_TIMESTAMP)
+	$(call assert-set,CF_BUILD_TRIGGER)
+
+	@$(MAKE) slack/notify/build \
+		ACTION=$(PIPELINE_ACTION) \
+		NAME=$(CHART_NAME) \
+		BUILD_URL=$(CF_BUILD_URL) \
+		REPO_OWNER=$(CF_REPO_OWNER) \
+		REPO_NAME=$(CF_REPO_NAME) \
+		BUILD_TIMESTAMP=$(shell expr $(CF_BUILD_TIMESTAMP) / 1000) \
+		BUILD_TRIGGER=$(CF_BUILD_TRIGGER)
+
+codefresh/notify/slack/deploy:
+	$(call assert-set,PIPELINE_ENV)
+	$(call assert-set,PIPELINE_ACTION)
+	$(call assert-set,CHART_NAME)
+	$(call assert-set,CF_BUILD_URL)
+	$(call assert-set,CF_REPO_OWNER)
+	$(call assert-set,CF_REPO_NAME)
+	$(call assert-set,CF_BUILD_TIMESTAMP)
+	$(call assert-set,CF_BUILD_TRIGGER)
+
+	@$(MAKE) slack/notify/deploy \
+		ACTION=$(PIPELINE_ACTION) \
+		NAME=$(CHART_NAME) \
+		ENVIRONMENT=$(PIPELINE_ENV) \
+		BUILD_URL=$(CF_BUILD_URL) \
+		REPO_OWNER=$(CF_REPO_OWNER) \
+		REPO_NAME=$(CF_REPO_NAME) \
+		BUILD_TIMESTAMP=$(shell expr $(CF_BUILD_TIMESTAMP) / 1000) \
+		BUILD_TRIGGER=$(CF_BUILD_TRIGGER)
+
+codefresh/notify/slack/deploy/exposed:
```

Comment:
rename to `success`?

Reply:
other options
- `endpoint`
- `webapp`
- `url`

Reply:
Done